app/views/how-tos/build-basic-prototype/link-pages-together.html

```diff
@@ -128,5 +130,10 @@ <h2 id="link-{{exampleCheckAnswers.url}}-to-your-{{exampleConfirmation.url}}-pag
     and select <strong>Accept and send registration</strong> to check the button works.
   </p>
 
+  {{ warningCallout({
```

Comment:
Thanks for adding this. Don't think this should be a warning callout though, perhaps inset text will do?

Reply:
changed to inset text